### PR TITLE
feat(client): link Arabic new articles to freeCodeCamp searchbar

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

ref #48101

### CODE REVIEW

File: client/src/utils/algolia-locale-setup.ts [hunk 1]
```diff
@@ -42,6 +42,10 @@ const algoliaIndices = {
   german: {
     name: 'news',
     searchPage: 'https://www.freecodecamp.org/news/search/'
+  },
+  arabic: {
+    name: 'news',
+    searchPage: 'https://www.freecodecamp.org/arabic/news/search/'
```
Comment (naomi-lgbt) [inline client/src/utils/algolia-locale-setup.ts:48]:
```suggestion
    searchPage: 'https://www.freecodecamp.org/news/search/'
``

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
How many articles do we have on the Arabic news instance now?

I'll cc @scissorsneedfoodtoo and @RafaelDavisH for the specific threshold, but we may want to start this pointed to the english instance?

(Edited because /news/search returns two articles apparently)

**Review by naomi-lgbt — DISMISSED** [review]
Should be good to go.

**Comment by scissorsneedfoodtoo** [comment]
Just checked on Algolia and there are currently 90 published articles in Arabic. It should be fine to set the index to `news-ar` and the search page to `https://www.freecodecamp.org/arabic/news/search/` here.

I'm not sure why visiting https://www.freecodecamp.org/arabic/news/search/ only returns two articles. The Urdu search page only returns one, rather than all articles like it should. I can look more into that.

Regarding the threshold, for News we've just been using the correct index for each publication when it launches. We generally require that there are about around 25 published articles before launching a publication. That won't lead to a whole lot of hits, but it's a decent amount for a new publication.

In the past, the threshold has only applied to the search bar placeholder. When a publication first launches we use a general label like "Search our tutorials". But when a publication hits 100+ articles, we switch over to a numbered placeholder like "Search 100+ tutorials".

**Review by scissorsneedfoodtoo — APPROVED** [review]
Everything LGTM 👍

**Review by sidemt — APPROVED** [review]
LGTM
(For future reference: Other configs related to enabling a language is in #48128 )

**Pull request merged**